### PR TITLE
Add Aggregation Layer to glossary

### PR DESCRIPTION
**Pull request by pyaillet**
Adding more terms: #5993

### CODE REVIEW

File: content/en/docs/reference/glossary/aggregation-layer.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
+---
+title: Aggregation Layer
+id: aggregation-layer
+date: 2018-10-08
+full_link: /docs/concepts/extend-kubernetes/api-extension/apiserver-aggregation/
+short_description: >
+  A mechanism to extend Kubernetes by aggregating the existing API Server with additional Kubernetes-style APIs servers
```
Comment (tengqm) [inline content/en/docs/reference/glossary/aggregation-layer.md]: Not sure if "APIs servers" is good English. Maybe we should say "API servers" instead?

**Comment by tengqm** [comment]
@Bradamant3 Mind take a look at this? It would be good to have native speakers to review PRs about glossary.

File: content/en/docs/reference/glossary/aggregation-layer.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+tags:
+- architecture
+- extension
+- operation
+---
+ A mechanism used to extend Kubernetes by aggregating the existing core APIs with additional Kubernetes-style APIs.
+
+<!--more--> 
+
+This mechanism must be activated on the {{< glossary_tooltip text="Kubernetes API Server" term_id="kube-apiserver" >}}, then users can add APIService objects, which “claim” an URL path in the Kubernetes API.
```
Comment (kbhawkey) [inline content/en/docs/reference/glossary/aggregation-layer.md]: Is the extension layer, on the kube-apiserver, activated by registering APIService objects?

The aggregation layer remains inactive until an APIService object is registered. When an APIService object is registered, the new URL path for this API is claimed or activated in the Kubernetes API.

File: content/en/docs/reference/glossary/aggregation-layer.md [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+tags:
+- architecture
+- extension
+- operation
+---
+ A mechanism used to extend Kubernetes by aggregating the existing core API with additional Kubernetes-style APIs.
+
+<!--more--> 
+
+This mechanism must be activated on the {{< glossary_tooltip text="Kubernetes API Server" term_id="kube-apiserver" >}}, then users can add APIService objects, which “claim” an URL path in the Kubernetes API.
```
Comment (zacharysarah) [inline content/en/docs/reference/glossary/aggregation-layer.md]: Replace with:
```
When you've configured the {{< glossary_tooltip text="Kubernetes API Server" term_id="kube-apiserver" >}} to [support additional APIs](https://kubernetes.io/docs/tasks/access-kubernetes-api/configure-aggregation-layer/), you can add `APIService` objects to "claim" a URL path in the Kubernetes API.
```
So:
> When you've configured the {{< glossary_tooltip text="Kubernetes API Server" term_id="kube-apiserver" >}} to [support additional APIs](https://kubernetes.io/docs/tasks/access-kubernetes-api/configure-aggregation-layer/), you can add `APIService` objects to "claim" a URL path in the Kubernetes API.

Comment (pyaillet) [reply content/en/docs/reference/glossary/aggregation-layer.md]: Thank you for the feedback, I'll make the adjustments.
I plan to propose some more additions to the glossary, I'll make sure to follow your guidelines.

**Pull request merged**